slack.php

```diff
@@ -87,6 +86,9 @@
     }
 
     $config = array_merge($defaultConfig, (array) $newConfig);
+    if(isset($newConfig['attachmentCustom'])){
```

Comment:
Please, fix code style

Reply:
What does you mean? 
`attachment-custom` ?
`attachment_custom` ?

Reply:
Spaces: ` if (isset($newConfig['attachmentCustom'])) {`

Reply:
@antonmedv fixed